### ISSUE TITLE
Is there any new release version after OpenTSDB 2.4.0？

### ISSUE THREAD

**Issue by plainee** [OpenTSDB/opentsdb#2071]
Is there any new release version after OpenTSDB 2.4.0？Last release is 2 years ago.

**Comment by manolama** [comment]
No, sorry for the wait. We've started on 2.5.0 but in the mean time we have some patches in the `master` branch of the Git repo that are fixes for 2.4. We'll have a 2.4.1 by the end of April.

**Comment by pengmq1** [comment]
> No, sorry for the wait. We've started on 2.5.0 but in the mean time we have some patches in the `master` branch of the Git repo that are fixes for 2.4. We'll have a 2.4.1 by the end of April.

Hi, manolama
What do you think about 3.0? Is any progress of 3.0 branch release?
I have downloaded code of 3.0, but it is not easily to use and run.
Recent commits are almost committed yourself. Does anyone else join 3.0? can i help contribute 3.0

**Comment by manolama** [comment]
@pengmq1 I'm re-working the write path now and getting that cleaned up and tested in production. Once that's done, I'll try and not let "perfection be the enemy of the good" and cut a release. The rest of the Yahoo monitoring team has been contributing and we recently OSSd some of our other components.
I've updated http://opentsdb.net/docs/3x/build/html/installation.html with some information that should help get 3.x running easier and if you would like to contribute, do a search for `TODO`s in the code. There are a ton :)
